ee/clickhouse/models/event.py

```diff
@@ -95,6 +156,7 @@ class ClickhouseEventSerializer(serializers.Serializer):
     person = serializers.SerializerMethodField()
     elements = serializers.SerializerMethodField()
     elements_chain = serializers.SerializerMethodField()
+    person_uuid = serializers.SerializerMethodField()
```

Comment:
Would it make sense to document the semantics of this here - so future generations can easily grasp the difference between uuids distinct_ids and person_ids?